migration/event-tap.c

```diff
@@ -612,9 +612,9 @@ int event_tap_flush(void* cb, void *opaque)
         ret = event_tap_flush_one();
     } while (ret == 0);
 
-    if (pending_bdrv_request == 0) {
```

Comment:
Why comment out this part?

Reply:
if no comment out this part, it would occur segmentation fault.